lava_test_plans/projects/lkft-android/devices/variables/dragonboard-845c.yaml

```diff
@@ -0,0 +1,7 @@
+DEVICE_TYPE: dragonboard-845c
+LAVA_JOB_PRIORITY: 25
+PROJECT: projects/lkft-android/
+TUXSUITE_BAKE_VENDOR_DOWNLOAD_URL: unknown
+BUILD_REFERENCE_IMAGE_GZ_URL: unknown
+LKFT_BUILD_CONFIG: unknown
+
```

Comment:
no need for this extra newline.
please remove

Reply:
ok @roxell